modules/integration_aws-backup/conf/03-aws-backup-check.yaml

```diff
@@ -0,0 +1,26 @@
+module: AWS Backup
+name: Check
+id: backup
+
+transformation: ".min(over='23h')"
+aggregation: true
+filtering: "filter('namespace', 'AWS/Backup')"
+value_unit: "count"
+
+signals:
+  created:
+    metric: NumberOfBackupJobsCreated
+    extrapolation: zero
+    rollup: sum
+  completed:
+    metric: NumberOfBackupJobsCompleted
+    extrapolation: zero
+    rollup: sum
+  signal:
+    formula: created - completed
+rules:
+  critical:
```

Comment:
you must provide a description because the detector's name only `Check` does not allow to understand the alert

Reply:
Addressed: https://github.com/claranet/terraform-signalfx-detectors/pull/457/files#diff-97fb0cb62ea20c06ac86dc8b101a21c2df8fdf968011a6584eb3a34669d95c4dR2